### PR TITLE
Fix/ABZ-34558 searchPath with macro fix

### PR DESCRIPTION
**Pull request by crdandoukho**

### CODE REVIEW

File: file-share-service-desktop-client/Modules/Admin/JobViewModels/NewBatchJobViewModel.cs [hunk 1]
```diff
@@ -351,17 +351,17 @@ private void ValidateViewModel()
             {
                 foreach (var file in Files)
                 {
-                    string? directory = Path.GetDirectoryName(file.RawSearchPath);
+                    string? directory = Path.GetDirectoryName(file.SearchPath);
 
                     if (string.IsNullOrWhiteSpace(directory))
                     {
-                        job.ErrorMessages.Add($"Directory not specified or invalid directory specified - '{file.RawSearchPath}'");
+                        job.ErrorMessages.Add($"Directory not specified or invalid directory specified - '{file.SearchPath}'");
```
Comment (rockydevnet) [inline file-share-service-desktop-client/Modules/Admin/JobViewModels/NewBatchJobViewModel.cs:358]: Could it be useful here to indicate both the raw and macro-expanded paths here? Raw helps the user find the right bit in the config file, expanded helps them see what's actually happening on disc?

Comment (crdandoukho) [reply file-share-service-desktop-client/Modules/Admin/JobViewModels/NewBatchJobViewModel.cs]: As requested, I have adjusted the error messages to include both the raw and expanded paths

File: Tests/UnitTests/FileShareService.DesktopClientTests/Modules/Admin/NewBatchJobViewModelTests.cs [hunk 2]
```diff
@@ -474,10 +477,13 @@ public void TestFileSearchWithMacroInDirectory(string directoryMacro, string exp
...
 
-
             Assert.AreEqual(2, vm.Files.SelectMany(f => f.Files).Count());
-            CollectionAssert.AreEqual(new[] { file1FullFileName, file2FullFileName },
+            CollectionAssert.AreEqual(new[] { filesA.expandedFilePath, filesB.expandedFilePath },
                 vm.Files.SelectMany(f => f.Files.Select(fi => fi.FullName)));
+
+            vm.ExcecuteJobCommand.CanExecute();
+
+            Assert.IsEmpty(vm.ValidationErrors);
```
Comment (barrie-cooper) [inline Tests/UnitTests/FileShareService.DesktopClientTests/Modules/Admin/NewBatchJobViewModelTests.cs:486]: I think this should do the same thing:
```suggestion
            Assert.IsTrue(vm.ExcecuteJobCommand.CanExecute());
```

Comment (crdandoukho) [reply Tests/UnitTests/FileShareService.DesktopClientTests/Modules/Admin/NewBatchJobViewModelTests.cs]: I agree. Thanks

**Review by barrie-cooper — APPROVED** [review]
LGTM

**Review by rockydevnet — COMMENTED** [review]
LGTM

**Pull request merged**